### ISSUE TITLE
TCHAR* <-> UTF8/ANSI macros used incorrectly; causing strings to go out of scope

### ISSUE THREAD

**Issue by sabhiram** [20tab/UnrealEnginePython#336]
Please see this commit on our own fork. 
https://github.com/recogni/UnrealEnginePython/commit/a1431aaf5bfbf641072b32f9a4e572ba3900045f

If you have questions, ping me. You can check the source code for the UE engine (StringConv.h) which describes the short-lived-ness of these macros.  Feel free to cherry-pick or re-implement. 

If I am super bored later, Ill send you a patch w/ our custom stuff stripped out.

**Comment by rdeioris** [comment]
Thanks a lot, the explanation is super clear, i will fix it soon.

**Comment by sabhiram** [comment]
Also, I know you mentioned before about implementing the `RunFile` variants with multiple args.  I took a shot at this and am using it to some decent success in our custom application.  Feel free to pull from here if you want: 
https://github.com/recogni/UnrealEnginePython/commit/be9c8a84910601504f9c54371ee0dd80fe668cc0

Some things I am planning on adding - once the `py.exec` (or in my case `py.exec_args`) is done, it would be nice if we could return the result of the exec (if any) to the caller. Maybe its overkill, but just thoughts :)
